mycity/mycity/utilities/csv_utils.py

```diff
@@ -5,11 +5,16 @@
 
 import collections
 import logging
+import typing
+
+from mycity.utilities.common_types import StrDict
 
 logger = logging.getLogger(__name__)
 
+NT = typing.TypeVar("NT", bound=tuple)
```

Comment:
Nitpick: Maybe return collections.namedtuple instead of this business?

Reply:
Done